pkg/controllers/disruption/singlenodeconsolidation.go

```diff
@@ -43,12 +43,11 @@ type SingleNodeConsolidation struct {
 }
 
 func NewSingleNodeConsolidation(consolidation consolidation) *SingleNodeConsolidation {
-	s := &SingleNodeConsolidation{
+	return &SingleNodeConsolidation{
 		consolidation:             consolidation,
 		PreviouslyUnseenNodePools: sets.New[string](),
+		Validator:                 NewSingleConsolidationValidator(consolidation),
```

Comment:
Ended up making a single and multi node validator because the consolidation type label on a metric represents `single` and `multi` and not `underutilized`.